src/hubot.d.ts

```diff
@@ -41,13 +41,28 @@ declare module "hubot" {
 
   }
 
-  interface IRobot {
+  interface IHubotLogger {
+    log(levelStr: string, args: any[]);
+    error(...msg: any[]);
+    emergency(...msg: any[]);
+    alert(...msg: any[]);
+    critical(...msg: any[]);
+    warning(...msg: any[]);
+    notice(...msg: any[]);
+    info(...msg: any[]);
+    debug(...msg: any[]);

```

Comment:
I'm really just being concise so we can contribute these back to [Definitely Typed](https://github.com/DefinitelyTyped/DefinitelyTyped).